README.md

```diff
@@ -68,10 +68,10 @@ if __name__ == '__main__':
 To use dashboards hosted by a remote computer, open an SSH tunnel to the remote computer and forward the port where the dashboard is hosted. For example, connect to the remote computer with
 
 ```bash
-ssh username@remote.server.org -L 8888:localhost:8888
+ssh username@remote.server.org -L 8890:localhost:8888
 ```
 
-to forward port 8888 on the host to port 8888 on your local computer.
+to forward port 8888 on the host to port 8890 on your local computer.
```

Comment:
@tcmoore3 It seems that you want to emphasize which port is which. However, I checked and it seems to be written backwards ([source](https://www.ssh.com/ssh/tunneling/example)). The format is `remotehost:remoteport:localhost:localport`, where `remotehost` is implied, so we're left with `remoteport:localhost:localport`. I suggest the following change:

```
ssh username@remote.server.org -L 8888:localhost:8890

to forward port 8888 on the remote host to port 8890 on your local computer.
```

Reply:
^ I'm wrong.